src/tests/electron/tests/automated-checks-view.test.ts

```diff
@@ -18,9 +21,9 @@ describe('AutomatedChecksView', () => {
             'beforeEach',
         );
         app = await createApplication({ suppressFirstTimeDialog: true });
-        const resultsView = await app.openResultsView();
-        await resultsView.waitForScreenshotViewVisible();
-        cardsView = resultsView.createCardsViewController();
+        // resultsView = await app.openResultsView();
```

Comment:
Are these meant to be commented out?

Reply:
Yes, I was trying to test something out. I've uncommented them